src/interfaces/widget.ts

```diff
@@ -133,8 +133,15 @@ export type WidgetFormField = Extract<WidgetField, WidgetFormFieldBase>
 
 export type WidgetListField = Extract<WidgetField, WidgetListFieldBase>
 
+/* description for a block structure containing fields on a form */
+export interface WidgetBlock {
```

Comment:
It seems that widget blocks are already defined by `WidgetFieldBlock` interface: https://github.com/tesler-platform/tesler-ui/blob/056ae71c4ae3313e5b87b96d0c8dd2d1ddd541e6/src/interfaces/widget.ts#L195-L201
Though it could definitely use some JSDoc with properties description.